src/main/java/com/healthapp/project/Main.java

```diff
@@ -2,30 +2,25 @@
 
 import com.healthapp.project.DAO.DataSourceConfig;
 import com.healthapp.project.Service.UserService;
+import javax.sql.DataSource;
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
 import org.springframework.context.ApplicationContext;
 import org.springframework.context.annotation.AnnotationConfigApplicationContext;
-import org.springframework.jdbc.core.JdbcTemplate;
-
-import javax.sql.DataSource;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.stereotype.Component;
 
 @SpringBootApplication
+@ComponentScan(basePackages = {"com.healthapp.project"})
 public class Main {
 
-
   public static void main(String[] args) {
-   SpringApplication.run(Main.class, args);
 
-   ApplicationContext context = new AnnotationConfigApplicationContext(DataSourceConfig.class);
-      DataSource dataSource = context.getBean(DataSource.class);
-     UserService userService = new UserService(dataSource);
-     System.out.print(userService.isDBConnected());
+    ApplicationContext context = SpringApplication.run(Main.class, args);
 
-    System.out.println(System.getenv("DB_URL_TEST"));
-    System.out.println(System.getenv("DB_USERNAME_TEST"));
-    System.out.println(System.getenv("DB_PASSWORD_TEST"));
 
+    UserService userService = context.getBean(UserService.class);
 
+  System.out.print(userService.isDBConnected());
   }
 }
```

Comment:
For me to be able to test the connection, i wanted to print the results to the console. In order to make UserService available to me in Main, i pulled its bean, spring injected the necessary dependencies and voila. No need to say 
UserService userservice = new Userservice(datasource)... none of that.